Sources/Fuzzilli/Core/ProgramBuilder.swift

```diff
@@ -388,43 +393,90 @@ public class ProgramBuilder {
     /// Append a splice from another program.
     public func splice(from program: Program, at index: Int) {
         var idx = index
-        
+
+        // As programs are ref counted, we create copy such that we do not
+        // modify the one in the corpus.
+        let workingCopy = program.copy()
```

Comment:
Nit: maybe rename this to e.g. `source`? and maybe change the comment to something like `The input rewiring algorithm modifies the source program, thus we make a copy of it here` to make it clear who modifies the program?

Reply:
👍